[libspatialite] Build error

Package: libspatialite[core,freexl,geocallbacks]:x64-osx-dynamic -> 5.1.0

**Host Environment**

- Host: x64-osx
- Compiler: AppleClang 14.0.0.14000029
-    vcpkg-tool version: 2023-08-09-9990a4c9026811a312cb2af78bf77f3d9d288416
    vcpkg-scripts version: 424ed5e67 2023-09-11 (7 days ago)

**To Reproduce**

`vcpkg install qgis[process,gui,desktop]:x64-osx-dynamic --recurse`
**Failure logs**

```
-- Using cached libspatialite-5.1.0.tar.gz.
-- Cleaning sources at /Users/mkuhn/dev/vcpkg/buildtrees/libspatialite/src/atialite-5-5796f3d8f9.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source /Users/mkuhn/dev/vcpkg/downloads/libspatialite-5.1.0.tar.gz
-- Applying patch fix-makefiles.patch
-- Applying patch fix-linux-configure.patch
-- Applying patch gaiaconfig-msvc.patch
-- Applying patch fix-mingw.patch
-- Applying patch fix-utf8-source.patch
-- Using source at /Users/mkuhn/dev/vcpkg/buildtrees/libspatialite/src/atialite-5-5796f3d8f9.clean
-- Getting CMake variables for x64-osx-dynamic-rel
-- Generating configure for x64-osx-dynamic
-- Finished generating configure for x64-osx-dynamic
-- Configuring x64-osx-dynamic-rel
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:112 (message):
    Command failed: /bin/bash -c "V=1 ./../src/atialite-5-5796f3d8f9.clean/configure  \"--target=macosx\" \"--enable-freexl\" \"--disable-gcp\" \"--enable-geocallbacks\" \"--disable-rttopo\" \"--disable-examples\" \"--disable-minizip\" \"--disable-silent-rules\" \"--verbose\" \"--enable-shared\" \"--disable-static\" \"LIBS=-L/Users/mkuhn/dev/vcpkg/installed/x64-osx-dynamic/lib/pkgconfig/../../lib -lgeos_c -lxml2 -lproj -lsqlite3 -lz -lfreexl -lm $LIBS\" \"--prefix=/Users/mkuhn/dev/vcpkg/installed/x64-osx-dynamic\" \"--bindir=\\${prefix}/tools/libspatialite/bin\" \"--sbindir=\\${prefix}/tools/libspatialite/sbin\" \"--libdir=\\${prefix}/lib\" \"--mandir=\\${prefix}/share/libspatialite\" \"--docdir=\\${prefix}/share/libspatialite\" \"--datarootdir=\\${prefix}/share/libspatialite\""
    Working Directory: /Users/mkuhn/dev/vcpkg/buildtrees/libspatialite/x64-osx-dynamic-rel
    Error code: 77
    See logs for more information:
      /Users/mkuhn/dev/vcpkg/buildtrees/libspatialite/config-x64-osx-dynamic-rel-config.log
      /Users/mkuhn/dev/vcpkg/buildtrees/libspatialite/config-x64-osx-dynamic-rel-out.log
      /Users/mkuhn/dev/vcpkg/buildtrees/libspatialite/config-x64-osx-dynamic-rel-err.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_configure_make.cmake:838 (vcpkg_execute_required_process)
  ports/libspatialite/portfile.cmake:179 (vcpkg_configure_make)
  scripts/ports.cmake:147 (include)



```
<details><summary>/Users/mkuhn/dev/vcpkg/buildtrees/libspatialite/config-x64-osx-dynamic-rel-config.log</summary>

```
This file contains any messages produced by compilers while
running configure, to aid debugging if configure makes a mistake.

It was created by libspatialite configure 5.1.0, which was
generated by GNU Autoconf 2.71.  Invocation command line was

  $ ./../src/atialite-5-5796f3d8f9.clean/configure --target=macosx --enable-freexl --disable-gcp --enable-geocallbacks --disable-rttopo --disable-examples --disable-minizip --disable-silent-rules --verbose --enable-shared --disable-static 'LIBS=-L/Users/mkuhn/dev/vcpkg/installed/x64-osx-dynamic/lib/pkgconfig/../../lib -lgeos_c -lxml2 -lproj -lsqlite3 -lz -lfreexl -lm ' --prefix=/Users/mkuhn/dev/vcpkg/installed/x64-osx-dynamic '--bindir=${prefix}/tools/libspatialite/bin' '--sbindir=${prefix}/tools/libspatialite/sbin' '--libdir=${prefix}/lib' '--mandir=${prefix}/share/libspatialite' '--docdir=${prefix}/share/libspatialite' '--datarootdir=${prefix}/share/libspatialite'

## --------- ##
## Platform. ##
## --------- ##

hostname = Matthiass-Air
uname -m = x86_64
uname -r = 21.6.0
uname -s = Darwin
uname -v = Darwin Kernel Version 21.6.0: Thu Jul  6 22:18:26 PDT 2023; root:xnu-8020.240.18.702.13~1/RELEASE_X86_64

/usr/bin/uname -p = i386
/bin/uname -X     = unknown

/bin/arch              = unknown
/usr/bin/arch -k       = unknown
/usr/convex/getsysinfo = unknown
/usr/bin/hostinfo      = Mach kernel version:
	 Darwin Kernel Version 21.6.0: Thu Jul  6 22:18:26 PDT 2023; root:xnu-8020.240.18.702.13~1/RELEASE_X86_64
Kernel configured for up to 4 processors.
2 processors are physically available.
4 processors are logically available.
Processor type: x86_64h (Intel x86-64h Haswell)
Processors active: 0 1 2 3
Primary memory available: 8.00 gigabytes
Default processor set: 518 tasks, 2030 threads, 4 processors
Load average: 2.28, Mach factor: 1.79
/bin/machine           = unknown
/usr/bin/oslevel       = unknown
/bin/universe          = unknown

PATH: /usr/local/bin/
PATH: /usr/local/sbin/
PATH: /usr/local/bin/
PATH: /usr/bin/
PATH: /bin/
PATH: /usr/sbin/
PATH: /sbin/
PATH: /Users/mkuhn/dev/vcpkg/downloads/tools/ninja/1.10.2-osx/
PATH: /usr/local/bin/


## ----------- ##
## Core tests. ##
## ----------- ##

configure:2997: looking for aux files: config.guess config.sub ltmain.sh compile missing install-sh
configure:3010:  trying ./../src/atialite-5-5796f3d8f9.clean/
configure:3039:   ./../src/atialite-5-5796f3d8f9.clean/config.guess found
configure:3039:   ./../src/atialite-5-5796f3d8f9.clean/config.sub found
configure:3039:   ./../src/atialite-5-5796f3d8f9.clean/ltmain.sh found
configure:3039:   ./../src/atialite-5-5796f3d8f9.clean/compile found
configure:3039:   ./../src/atialite-5-5796f3d8f9.clean/missing found
configure:3021:   ./../src/atialite-5-5796f3d8f9.clean/install-sh found
configure:3176: checking for a BSD-compatible install
configure:3249: result: /usr/bin/install -c
configure:3260: checking whether build environment is sane
configure:3315: result: yes
configure:3474: checking for a race-free mkdir -p
configure:3518: result: ./../src/atialite-5-5796f3d8f9.clean/install-sh -c -d
configure:3525: checking for gawk
configure:3560: result: no
configure:3525: checking for mawk
configure:3560: result: no
configure:3525: checking for nawk
configure:3560: result: no
configure:3525: checking for awk
configure:3546: found /usr/bin/awk
configure:3557: result: awk
configure:3568: checking whether make sets $(MAKE)
configure:3591: result: yes
configure:3621: checking whether make supports nested variables
configure:3639: result: yes
configure:3771: checking whether to enable maintainer-specific portions of Makefiles
configure:3781: result: no
configure:3851: checking whether make supports the include directive
configure:3866: make -f confmf.GNU && cat confinc.out
this is the am__doit target
configure:3869: $? = 0
configure:3888: result: yes (GNU style)
configure:3964: checking for gcc
configure:3996: result: /Library/Developer/CommandLineTools/usr/bin/cc -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk -DNDEBUG -fPIC -mmacosx-version-min=12.6 -O3
configure:4349: checking for C compiler version
configure:4358: /Library/Developer/CommandLineTools/usr/bin/cc -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk -DNDEBUG -fPIC -mmacosx-version-min=12.6 -O3 --version >&5
Apple clang version 14.0.0 (clang-1400.0.29.202)
Target: x86_64-apple-darwin21.6.0
Thread model: posix
InstalledDir: /Library/Developer/CommandLineTools/usr/bin
configure:4369: $? = 0
configure:4358: /Library/Developer/CommandLineTools/usr/bin/cc -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk -DNDEBUG -fPIC -mmacosx-version-min=12.6 -O3 -v >&5
Apple clang version 14.0.0 (clang-1400.0.29.202)
Target: x86_64-apple-darwin21.6.0
Thread model: posix
InstalledDir: /Library/Developer/CommandLineTools/usr/bin
clang: warning: argument unused during compilation: '-isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk' [-Wunused-command-line-argument]
clang: warning: argument unused during compilation: '-D NDEBUG' [-Wunused-command-line-argument]
clang: warning: argument unused during compilation: '-fPIC' [-Wunused-command-line-argument]
clang: warning: argument unused during compilation: '-mmacosx-version-min=12.6' [-Wunused-command-line-argument]
clang: warning: argument unused during compilation: '-O3' [-Wunused-command-line-argument]
configure:4369: $? = 0
configure:4358: /Library/Developer/CommandLineTools/usr/bin/cc -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk -DNDEBUG -fPIC -mmacosx-version-min=12.6 -O3 -V >&5
clang: error: argument to '-V' is missing (expected 1 value)
clang: error: no input files
configure:4369: $? = 1
configure:4358: /Library/Developer/CommandLineTools/usr/bin/cc -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk -DNDEBUG -fPIC -mmacosx-version-min=12.6 -O3 -qversion >&5
clang: error: unknown argument '-qversion'; did you mean '--version'?
clang: error: no input files
configure:4369: $? = 1
configure:4358: /Library/Developer/CommandLineTools/usr/bin/cc -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk -DNDEBUG -fPIC -mmacosx-version-min=12.6 -O3 -version >&5
clang: error: unknown argument '-version'; did you mean '--version'?
clang: error: no input files
configure:4369: $? = 1
configure:4389: checking whether the C compiler works
configure:4411: /Library/Developer/CommandLineTools/usr/bin/cc -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk -DNDEBUG -fPIC -mmacosx-version-min=12.6 -O3 -fPIC -mmacosx-version-min=12.6 -O3 -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk -DNDEBUG -L/Users/mkuhn/dev/vcpkg/installed/x64-osx-dynamic/lib -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk -mmacosx-version-min=12.6 conftest.c -L/Users/mkuhn/dev/vcpkg/installed/x64-osx-dynamic/lib/pkgconfig/../../lib -lgeos_c -lxml2 -lproj -lsqlite3 -lz -lfreexl -lm  >&5
configure:4415: $? = 0
configure:4465: result: yes
configure:4468: checking for C compiler default output file name
configure:4470: result: a.out
configure:4476: checking for suffix of executables
configure:4483: /Library/Developer/CommandLineTools/usr/bin/cc -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk -DNDEBUG -fPIC -mmacosx-version-min=12.6 -O3 -o conftest -fPIC -mmacosx-version-min=12.6 -O3 -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk -DNDEBUG -L/Users/mkuhn/dev/vcpkg/installed/x64-osx-dynamic/lib -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk -mmacosx-version-min=12.6 conftest.c -L/Users/mkuhn/dev/vcpkg/installed/x64-osx-dynamic/lib/pkgconfig/../../lib -lgeos_c -lxml2 -lproj -lsqlite3 -lz -lfreexl -lm  >&5
configure:4487: $? = 0
configure:4510: result: 
configure:4532: checking whether we are cross compiling
configure:4540: /Library/Developer/CommandLineTools/usr/bin/cc -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk -DNDEBUG -fPIC -mmacosx-version-min=12.6 -O3 -o conftest -fPIC -mmacosx-version-min=12.6 -O3 -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk -DNDEBUG -L/Users/mkuhn/dev/vcpkg/installed/x64-osx-dynamic/lib -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk -mmacosx-version-min=12.6 conftest.c -L/Users/mkuhn/dev/vcpkg/installed/x64-osx-dynamic/lib/pkgconfig/../../lib -lgeos_c -lxml2 -lproj -lsqlite3 -lz -lfreexl -lm  >&5
configure:4544: $? = 0
configure:4551: ./conftest
dyld[31214]: Library not loaded: '@rpath/libgeos_c.1.dylib'
  Referenced from: '/Users/mkuhn/dev/vcpkg/buildtrees/libspatialite/x64-osx-dynamic-rel/conftest'
  Reason: tried: '/usr/local/lib/libgeos_c.1.dylib' (no such file), '/usr/lib/libgeos_c.1.dylib' (no such file)
./../src/atialite-5-5796f3d8f9.clean/configure: line 4553: 31214 Abort trap: 6           ./conftest$ac_cv_exeext
configure:4555: $? = 134
configure:4562: error: in `/Users/mkuhn/dev/vcpkg/buildtrees/libspatialite/x64-osx-dynamic-rel':
configure:4564: error: cannot run C compiled programs.
If you meant to cross compile, use `--host'.
See `config.log' for more details

## ---------------- ##
## Cache variables. ##
## ---------------- ##

ac_cv_env_CCC_set=
ac_cv_env_CCC_value=
ac_cv_env_CC_set=set
ac_cv_env_CC_value='/Library/Developer/CommandLineTools/usr/bin/cc -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk -DNDEBUG -fPIC -mmacosx-version-min=12.6 -O3'
ac_cv_env_CFLAGS_set=set
ac_cv_env_CFLAGS_value='-fPIC -mmacosx-version-min=12.6 -O3'
ac_cv_env_CPPFLAGS_set=set
ac_cv_env_CPPFLAGS_value='-isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk -DNDEBUG'
ac_cv_env_CPP_set=
ac_cv_env_CPP_value=
ac_cv_env_CXXCPP_set=
ac_cv_env_CXXCPP_value=
ac_cv_env_CXXFLAGS_set=set
ac_cv_env_CXXFLAGS_value='-fPIC -mmacosx-version-min=12.6 -O3'
ac_cv_env_CXX_set=set
ac_cv_env_CXX_value=/Library/Developer/CommandLineTools/usr/bin/c++
ac_cv_env_LDFLAGS_set=set
ac_cv_env_LDFLAGS_value='-L/Users/mkuhn/dev/vcpkg/installed/x64-osx-dynamic/lib -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk -mmacosx-version-min=12.6'
ac_cv_env_LIBS_set=set
ac_cv_env_LIBS_value='-L/Users/mkuhn/dev/vcpkg/installed/x64-osx-dynamic/lib/pkgconfig/../../lib -lgeos_c -lxml2 -lproj -lsqlite3 -lz -lfreexl -lm '
ac_cv_env_LIBXML2_CFLAGS_set=
ac_cv_env_LIBXML2_CFLAGS_value=
ac_cv_env_LIBXML2_LIBS_set=
ac_cv_env_LIBXML2_LIBS_value=
ac_cv_env_LT_SYS_LIBRARY_PATH_set=
ac_cv_env_LT_SYS_LIBRARY_PATH_value=
ac_cv_env_PKG_CONFIG_LIBDIR_set=
ac_cv_env_PKG_CONFIG_LIBDIR_value=
ac_cv_env_PKG_CONFIG_PATH_set=set
ac_cv_env_PKG_CONFIG_PATH_value=/Users/mkuhn/dev/vcpkg/installed/x64-osx-dynamic/lib/pkgconfig/:/Users/mkuhn/dev/vcpkg/installed/x64-osx-dynamic/share/pkgconfig/
ac_cv_env_PKG_CONFIG_set=set
ac_cv_env_PKG_CONFIG_value=/usr/local/bin/pkg-config
ac_cv_env_SQLITE3_CFLAGS_set=
ac_cv_env_SQLITE3_CFLAGS_value=
ac_cv_env_SQLITE3_LIBS_set=
ac_cv_env_SQLITE3_LIBS_value=
ac_cv_env_build_alias_set=
ac_cv_env_build_alias_value=
ac_cv_env_host_alias_set=
ac_cv_env_host_alias_value=
ac_cv_env_target_alias_set=set
ac_cv_env_target_alias_value=macosx
ac_cv_path_install='/usr/bin/install -c'
ac_cv_prog_AWK=awk
ac_cv_prog_ac_ct_CC='/Library/Developer/CommandLineTools/usr/bin/cc -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk -DNDEBUG -fPIC -mmacosx-version-min=12.6 -O3'
ac_cv_prog_make_make_set=yes
am_cv_make_support_nested_variables=yes

## ----------------- ##
## Output variables. ##
## ----------------- ##

ACLOCAL='${SHELL} '\''/Users/mkuhn/dev/vcpkg/buildtrees/libspatialite/src/atialite-5-5796f3d8f9.clean/missing'\'' aclocal-1.16'
AMDEPBACKSLASH='\'
AMDEP_FALSE='#'
AMDEP_TRUE=''
AMTAR='$${TAR-tar}'
AM_BACKSLASH='\'
AM_DEFAULT_V='$(AM_DEFAULT_VERBOSITY)'
AM_DEFAULT_VERBOSITY='1'
AM_V='$(V)'
ANDROID_FALSE=''
ANDROID_TRUE=''
AR='/Library/Developer/CommandLineTools/usr/bin/ar'
AS=''
AUTOCONF='${SHELL} '\''/Users/mkuhn/dev/vcpkg/buildtrees/libspatialite/src/atialite-5-5796f3d8f9.clean/missing'\'' autoconf'
AUTOHEADER='${SHELL} '\''/Users/mkuhn/dev/vcpkg/buildtrees/libspatialite/src/atialite-5-5796f3d8f9.clean/missing'\'' autoheader'
AUTOMAKE='${SHELL} '\''/Users/mkuhn/dev/vcpkg/buildtrees/libspatialite/src/atialite-5-5796f3d8f9.clean/missing'\'' automake-1.16'
AWK='awk'
CC='/Library/Developer/CommandLineTools/usr/bin/cc -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk -DNDEBUG -fPIC -mmacosx-version-min=12.6 -O3'
CCDEPMODE=''
CFLAGS='-fPIC -mmacosx-version-min=12.6 -O3'
CPP=''
CPPFLAGS='-isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk -DNDEBUG'
CSCOPE='cscope'
CTAGS='ctags'
CXX='/Library/Developer/CommandLineTools/usr/bin/c++'
CXXCPP=''
CXXDEPMODE=''
CXXFLAGS='-fPIC -mmacosx-version-min=12.6 -O3'
CYGPATH_W='echo'
DEFS=''
DEPDIR='.deps'
DLLTOOL=''
DSYMUTIL=''
DUMPBIN=''
ECHO_C='\c'
ECHO_N=''
ECHO_T=''
EGREP=''
ENABLE_EXAMPLES_FALSE=''
ENABLE_EXAMPLES_TRUE=''
ENABLE_GEOPACKAGE_FALSE=''
ENABLE_GEOPACKAGE_TRUE=''
ETAGS='etags'
EXEEXT=''
FGREP=''
FILECMD=''
GEOSCONFIG=''
GEOS_CFLAGS=''
GEOS_LDFLAGS=''
GREP=''
INSTALL_DATA='${INSTALL} -m 644'
INSTALL_PROGRAM='${INSTALL}'
INSTALL_SCRIPT='${INSTALL}'
INSTALL_STRIP_PROGRAM='$(install_sh) -c -s'
LD='/Library/Developer/CommandLineTools/usr/bin/ld'
LDFLAGS='-L/Users/mkuhn/dev/vcpkg/installed/x64-osx-dynamic/lib -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk -mmacosx-version-min=12.6'
LIBOBJS=''
LIBS='-L/Users/mkuhn/dev/vcpkg/installed/x64-osx-dynamic/lib/pkgconfig/../../lib -lgeos_c -lxml2 -lproj -lsqlite3 -lz -lfreexl -lm '
LIBTOOL=''
LIBXML2_CFLAGS=''
LIBXML2_LIBS=''
LIPO=''
LN_S=''
LTLIBOBJS=''
LT_SYS_LIBRARY_PATH=''
MACOSX_FALSE=''
MACOSX_TRUE=''
MAINT='#'
MAINTAINER_MODE_FALSE=''
MAINTAINER_MODE_TRUE='#'
MAKEINFO='${SHELL} '\''/Users/mkuhn/dev/vcpkg/buildtrees/libspatialite/src/atialite-5-5796f3d8f9.clean/missing'\'' makeinfo'
MANIFEST_TOOL=''
MINGW_FALSE=''
MINGW_TRUE=''
MKDIR_P='./../src/atialite-5-5796f3d8f9.clean/install-sh -c -d'
MODULE_ONLY_FALSE=''
MODULE_ONLY_TRUE=''
NM='/Library/Developer/CommandLineTools/usr/bin/nm'
NMEDIT=''
OBJDUMP='/Library/Developer/CommandLineTools/usr/bin/objdump'
OBJEXT=''
OTOOL64=''
OTOOL=''
PACKAGE='libspatialite'
PACKAGE_BUGREPORT='a.furieri@lqt.it'
PACKAGE_NAME='libspatialite'
PACKAGE_STRING='libspatialite 5.1.0'
PACKAGE_TARNAME='libspatialite'
PACKAGE_URL=''
PACKAGE_VERSION='5.1.0'
PATH_SEPARATOR=':'
PKG_CONFIG='/usr/local/bin/pkg-config'
PKG_CONFIG_LIBDIR=''
PKG_CONFIG_PATH='/Users/mkuhn/dev/vcpkg/installed/x64-osx-dynamic/lib/pkgconfig/:/Users/mkuhn/dev/vcpkg/installed/x64-osx-dynamic/share/pkgconfig/'
RANLIB='/Library/Developer/CommandLineTools/usr/bin/ranlib'
SED=''
SET_MAKE=''
SHELL='/bin/sh'
SQLITE3_CFLAGS=''
SQLITE3_LIBS=''
STRIP='/Library/Developer/CommandLineTools/usr/bin/strip'
VERSION='5.1.0'
ac_ct_AR=''
ac_ct_CC='/Library/Developer/CommandLineTools/usr/bin/cc -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk -DNDEBUG -fPIC -mmacosx-version-min=12.6 -O3'
ac_ct_CXX=''
ac_ct_DUMPBIN=''
am__EXEEXT_FALSE=''
am__EXEEXT_TRUE=''
am__fastdepCC_FALSE=''
am__fastdepCC_TRUE=''
am__fastdepCXX_FALSE=''
am__fastdepCXX_TRUE=''
am__include='include'
am__isrc=' -I$(srcdir)'
am__leading_dot='.'
am__nodep='_no'
am__quote=''
am__tar='$${TAR-tar} chof - "$$tardir"'
am__untar='$${TAR-tar} xf -'
bindir='${prefix}/tools/libspatialite/bin'
build=''
build_alias=''
build_cpu=''
build_os=''
build_vendor=''
datadir='${datarootdir}'
datarootdir='${prefix}/share/libspatialite'
docdir='${prefix}/share/libspatialite'
dvidir='${docdir}'
exec_prefix='NONE'
host=''
host_alias=''
host_cpu=''
host_os=''
host_vendor=''
htmldir='${docdir}'
includedir='${prefix}/include'
infodir='${datarootdir}/info'
install_sh='${SHELL} /Users/mkuhn/dev/vcpkg/buildtrees/libspatialite/src/atialite-5-5796f3d8f9.clean/install-sh'
libdir='${prefix}/lib'
libexecdir='${exec_prefix}/libexec'
localedir='${datarootdir}/locale'
localstatedir='${prefix}/var'
mandir='${prefix}/share/libspatialite'
mkdir_p='$(MKDIR_P)'
oldincludedir='/usr/include'
pdfdir='${docdir}'
prefix='/Users/mkuhn/dev/vcpkg/installed/x64-osx-dynamic'
program_transform_name='s,x,x,'
psdir='${docdir}'
runstatedir='${localstatedir}/run'
sbindir='${prefix}/tools/libspatialite/sbin'
sharedstatedir='${prefix}/com'
sysconfdir='${prefix}/etc'
target_alias='macosx'

## ----------- ##
## confdefs.h. ##
## ----------- ##

/* confdefs.h */
#define PACKAGE_NAME "libspatialite"
#define PACKAGE_TARNAME "libspatialite"
#define PACKAGE_VERSION "5.1.0"
#define PACKAGE_STRING "libspatialite 5.1.0"
#define PACKAGE_BUGREPORT "a.furieri@lqt.it"
#define PACKAGE_URL ""
#define _LARGE_FILE 1
#define _FILE_OFFSET_BITS 64
#define _LARGEFILE_SOURCE 1
#define NDEBUG 1

configure: exit 77
```
</details>
<details><summary>/Users/mkuhn/dev/vcpkg/buildtrees/libspatialite/config-x64-osx-dynamic-rel-out.log</summary>

```
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for a race-free mkdir -p... ./../src/atialite-5-5796f3d8f9.clean/install-sh -c -d
checking for gawk... no
checking for mawk... no
checking for nawk... no
checking for awk... awk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking whether to enable maintainer-specific portions of Makefiles... no
checking whether make supports the include directive... yes (GNU style)
checking for gcc... /Library/Developer/CommandLineTools/usr/bin/cc -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk -DNDEBUG -fPIC -mmacosx-version-min=12.6 -O3
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... 
```
</details>
<details><summary>/Users/mkuhn/dev/vcpkg/buildtrees/libspatialite/config-x64-osx-dynamic-rel-err.log</summary>

```
configure: WARNING: unrecognized options: --enable-geocallbacks
configure: error: in `/Users/mkuhn/dev/vcpkg/buildtrees/libspatialite/x64-osx-dynamic-rel':
configure: error: cannot run C compiled programs.
If you meant to cross compile, use `--host'.
See `config.log' for more details
```
</details>

--- a/ports/geos/fix-exported-config.patch
+++ b/ports/geos/fix-exported-config.patch
@@ -66,11 +66,11 @@ index 6eff1eb14..8827f6ac6 100644
        ;;
      --static-clibs)
 -      echo -L${libdir} -lgeos_c -lgeos -lstdc++ -lm
-+      echo -L${libdir} -lgeos_c -lgeos -lstdc++ @EXTRA_LIBS@
++      echo -L${libdir} -lgeos_c -lgeos @EXTRA_LIBS@
        ;;
      --static-cclibs)
 -      echo -L${libdir} -lgeos -lstdc++ -lm
-+      echo -L${libdir} -lgeos -lstdc++ @EXTRA_LIBS@
++      echo -L${libdir} -lgeos @EXTRA_LIBS@
        ;;
      --cflags)
        echo -I${includedir}

--- a/ports/geos/vcpkg.json
+++ b/ports/geos/vcpkg.json
@@ -1,6 +1,7 @@
 {
   "name": "geos",
   "version": "3.11.3",
+  "port-version": 1,
   "description": "Geometry Engine Open Source",
   "homepage": "https://libgeos.org/",
   "license": "LGPL-2.1-only",

--- a/ports/libspatialite/portfile.cmake
+++ b/ports/libspatialite/portfile.cmake
@@ -19,7 +19,6 @@ vcpkg_check_features(OUT_FEATURE_OPTIONS unused
     FEATURES
         freexl          ENABLE_FREEXL
         gcp             ENABLE_GCP
-        geocallbacks    ENABLE_GEOCALLBACKS
         rttopo          ENABLE_RTTOPO
 )
 
@@ -38,9 +37,6 @@ if(VCPKG_TARGET_IS_WINDOWS AND NOT VCPKG_TARGET_IS_MINGW)
     endif()
     if(ENABLE_GCP)
         string(APPEND CL_FLAGS " /DENABLE_GCP")
-    endif()
-    if(NOT ENABLE_GEOCALLBACKS)
-        string(APPEND CL_FLAGS " /DOMIT_GEOCALLBACKS")
     endif()
     if(ENABLE_RTTOPO)
         string(APPEND CL_FLAGS " /DENABLE_RTTOPO")
@@ -184,10 +180,10 @@ else()
             ${TARGET_ALIAS}
             ${FREEXL_OPTION}
             ${GCP_OPTION}
-            ${GEOCALLBACKS_OPTION}
             ${RTTOPO_OPTION}
             "--disable-examples"
             "--disable-minizip"
+            "cross_compiling=yes" # avoid conftest rpath trouble
         OPTIONS_DEBUG
             "LIBS=${PKGCONFIG_LIBS_DEBUG} ${SYSTEM_LIBS}"
         OPTIONS_RELEASE

--- a/ports/libspatialite/vcpkg.json
+++ b/ports/libspatialite/vcpkg.json
@@ -1,14 +1,16 @@
 {
   "name": "libspatialite",
   "version": "5.1.0",
+  "port-version": 1,
   "description": "SpatiaLite is an open source library intended to extend the SQLite core to support fully fledged Spatial SQL capabilities.",
-  "homepage": "https://www.gaia-gis.it/gaia-sins/libspatialite-sources",
+  "homepage": "https://www.gaia-gis.it/fossil/libspatialite/index",
   "license": null,
   "dependencies": [
     "geos",
     "libiconv",
     {
       "name": "libxml2",
+      "default-features": false,
       "features": [
         "http"
       ]
@@ -28,8 +30,7 @@
     "zlib"
   ],
   "default-features": [
-    "freexl",
-    "geocallbacks"
+    "freexl"
   ],
   "features": {
     "freexl": {
@@ -40,9 +41,6 @@
     },
     "gcp": {
       "description": "Ground control points support. This feature reduces the license options to GPLv2+."
-    },
-    "geocallbacks": {
-      "description": "Geometry callbacks support."
     },
     "rttopo": {
       "description": "RTTOPO support. This feature reduces the license options to GPLv2+.",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -4834,7 +4834,7 @@
     },
     "libspatialite": {
       "baseline": "5.1.0",
-      "port-version": 0
+      "port-version": 1
     },
     "libspnav": {
       "baseline": "0.2.3",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -2914,7 +2914,7 @@
     },
     "geos": {
       "baseline": "3.11.3",
-      "port-version": 0
+      "port-version": 1
     },
     "geotrans": {
       "baseline": "3.9",

--- a/versions/g-/geos.json
+++ b/versions/g-/geos.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "41bb89a641e2fd76a21cdf259893ef3ab187aeaf",
+      "version": "3.11.3",
+      "port-version": 1
+    },
+    {
       "git-tree": "74ed7ac2ba209cbd984174705b2e3fc7fcf99e9a",
       "version": "3.11.3",
       "port-version": 0

--- a/versions/l-/libspatialite.json
+++ b/versions/l-/libspatialite.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "583ff9e938f0ca09b16f01fc5fb228b42dc13c7a",
+      "version": "5.1.0",
+      "port-version": 1
+    },
+    {
       "git-tree": "6c988a575680cb06fded62a1f63d43426d683dfd",
       "version": "5.1.0",
       "port-version": 0

